src/ai-bundle/src/AiBundle.php

```diff
@@ -95,6 +96,11 @@ public function loadExtension(array $config, ContainerConfigurator $container, C
     {
         $container->import('../config/services.php');
 
+        // COMMANDS
+        $builder->register('ai.command.chat', ChatCommand::class)
+            ->addTag('console.command')
+            ->setArgument(0, new Reference('ai.locator.agents'));
```

Comment:
The command is a service, Why is it defined here and not in `service.php` like any other service?

The service definition would use `tagged_locator` instead of the `ai.locator.agents` definition.

I don't understand why you need to manually populate the `ai.locator.agents` from `$builder->findTaggedServiceIds('ai.agent')`.

Reply:
Tried different things, but you are right